features/step_definitions/sriov.rb

```diff
@@ -0,0 +1,122 @@
+Given /^the sriov operator is running well$/ do
+  ensure_destructive_tagged
+  ensure_admin_tagged
+  step %Q/I switch to cluster admin pseudo user/
+  step %Q/evaluation of `cluster_version('version').channel.split('-')[1]` is stored in the :ocp_cluster_version clipboard/
+
+  unless project('openshift-sriov-network-operator').exists?
+    if cb.ocp_cluster_version.include? "4.[34]."
+      # create 4.3/4.4 namespaces
+      namespace_yaml ||= "#{ENV['BUSHSLICER_HOME']}/testdata/networking/sriov/namespace/43/ns.yaml"
```

Comment:
@zhaozhanqi please use the `I obtain the datafile` step

Reply:
@pruan-rht like the following steps?
```
  step %Q{I obtain test data file "networking/sriov/namespace/43/ns.yaml"}
  ns = YAML.load(@result[:response])
  namespace_yaml = @result[:abs_path]
```

Reply:
@zhaozhanqi you just need to call ```step %Q{I obtain test data file "networking/sriov/namespace/43/ns.yaml"}```.  After that execution, you can then reference `ns.yaml`

Reply:
@pruan-rht updated. thanks